equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/util/DateUtil.java

```diff
@@ -43,4 +47,18 @@ public static Date convertStringToSimpleDate(String dateToConvert) throws ParseE
 
         return new SimpleDateFormat(DATE_FORMAT_PATTERN, Locale.UK).parse(dateToConvert);
     }
+
+    public static String timeDifference(String startDateTime, String endDateTime, String format) {
```

Comment:
I would suggest rename to `calculateTimeDifference` or `getTimeDifference` for short. 
if you follow my previous comment about moving some constants to this class.
A good helper function would be `getTimeSpent(String timeReference)` and call this method with the start time constant passed by default

Reply:
modified. Done